api/server/server.go

```diff
@@ -22,14 +22,18 @@ import (
 	"github.com/textileio/powergate/deals"
```

Comment:
This file is mostly a bootstrapping point for creating the server. Apart from creating some extra things due to new features, I started extracting a bit of the logic of creating things to improve readability.

I wouldn't say it's in a perfect state, but better than before. I think what would enable a cleaner next step is grouping dependencies in the `Sever` struct instead of being flat, that would allow continuing creating new methods that would create that sub-parts of the Sever... which can have a `Close()` method to a more reasonable close-flow.